pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -70,8 +71,10 @@ const (
 	// were to continuously return a nextPageToken.
 	maxPages = 25
 
-	// Target Pool creation is limited to 200 instances.
 	maxTargetPoolCreateInstances = 200
+
+	// Max size of names accepted by the GCE api.
+	nameLengthLimit = 63
```

Comment:
or just `/` ? the UID should fit but is harder to debug (not a big deal though since we can make / redirect to uid for humans).